Marlin/ultralcd_impl_DOGM.h

```diff
@@ -67,33 +69,12 @@
   #define FONT_STATUSMENU_NAME FONT_MENU_NAME
 #endif
 
-#include "dogm_font_data_Marlin_symbols.h"   // The Marlin special symbols
-#define FONT_SPECIAL_NAME Marlin_symbols
+#include LANGUAGE_DATA_INCL(LCD_LANGUAGE)
+#define TALL_FONT_CORRECTION 1
```

Comment:
If `TALL_FONT_CORRECTION` is always defined - it can go away.
but it will hard to find a harmonic view then - fitting for Chinese and the other languages.